src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/Repository/MethodologyRepositoryTests.cs

```diff
@@ -26,6 +26,9 @@ public async Task CreateMethodologyForPublication()
                 Slug = "the-publication-slug"
             };
 
+            var userId = Guid.NewGuid();
+            var createdDate = DateTime.UtcNow.AddDays(-2);
```

Comment:
out of curiosity why are we passing -2 here?

Reply:
Totally arbitrary - just anything that isn't exactly now :)